templates/NOTES.txt

```diff
@@ -40,8 +40,8 @@ Ingress is enabled. You may access following endpoints:
 {{- end }}
 
 2. Login with the following credentials:
-username: {{ .Values.secrets.st2.username }}
-password: {{ .Values.secrets.st2.password }}
+username: {{ .Values.st2.username }}
+password: {{ .Values.st2.password }}
```

Comment:
Will it output the auto-generated password as well? if `Values.st2.password` is not initially set

Reply:
Good call. Hmm. This probably needs to do a `lookup` here.

Reply:
fixed